snap/snapcraft.yaml

```diff
@@ -25,6 +25,10 @@ platforms:
     build-for: [arm64]
 
 plugs:
+  sched-debugfs:
```

Comment:
```suggestion
  system-scheduler-debugfs:
```
To make it a bit clearer that this is a system plug. Maybe even `system-files-scheduler-debugfs`, so the user can guess the plug?

Reply:
But aren't most plugs system plugs? They provide capabilities to interact with the host system. I could rename it to `scheduler-debugfs` to make it clearer what it is about.

Reply:
Try if the following works:
```
sudo snap connect rt-tests:sched-debugfs
```

Reply:
It works!